src/mlpack/methods/ann/layer/atrous_convolution.hpp

```diff
@@ -257,6 +257,14 @@ class AtrousConvolution
   //! Modify the internal Padding layer.
   ann::Padding<>& Padding() { return padding; }
 
+  //! Get size of the weight matrix.
+  size_t WeightSize() const
+  {
+    return (outSize * inSize * kernelWidth * kernelHeight) +
+        outSize;
```

Comment:
```suggestion
    return (outSize * inSize * kernelWidth * kernelHeight) + outSize;
```
This is short enough to fit on one line. :+1: